client/src/variable.scss

```
@@ -47,4 +47,13 @@ $fz-medium-2: 2.4rem;
 
 $fz-large-1: 3.6rem;
 $fz-large-2: 4.1rem;
-$fz-large-3: 4.4rem;
\ No newline at end of file
+$fz-large-3: 4.4rem;
+
+// Variablen für Tags
+// a-tags color
+$ac: #777777;
+
+// Variablen für Font-Weights
+$fw-medium: 400;
+$fw-regular: 500;
+$fw-bold: 700;
```

Die variablen namen sind nicht ganz richtig.
```suggestion
$fw-regular: 400;
$fw-medium: 500;
$fw-bold: 700;
```
Für Referenz kannst du hier schauen: https://fonts.google.com/specimen/Roboto

---

done!